en/docs/get-started/quick-start-guide.md

```diff
@@ -10,96 +10,88 @@ This quick start guide walks you through the steps to create, develop, publish,
+Next, let's deploy this API and make it invokable.
  
 ### Step 2: Deploy the REST API 
 
-Let's deploy the API to make it invokable. Choreo maintains 3 environments by default: development and production. We will deploy the API to the development environment. 
+Choreo maintains two environments by default: development and production. Let's deploy the API to the development environment.
 
-1. Click **Deploy** from the left navigation. 
+1. Click **Deploy** in the left pane. 
 
-2. Click **Deploy**.
+2. Go to the **Build Area** card and click **Deploy**.
 
     ![Deploy API](../assets/img/get-started/deploy-rest-api.png){.cInlineImage-full}
 
-You have now successfully deployed your API to the development environment. It is now invokable. Let's test it and verify if it works.
+   This deploys the API to the development environment to make it invokable. 
+
+Now let's test the REST API to verify whether it works as expected.
 
 ### Step 3: Test the REST API 
 
-1. Click **Test** from the left navigation.
+1. Click **Test** in the left pane.
 
 2. Expand the **POST** method with the **/** resource path. 
     ![POST method](../assets/img/get-started/post-method.png){.cInlineImage-full}
 
 3. Click **Try it out**.
 
-4. Add a meaningful string to the request body. E.g.: `Welcome to Choreo`.
+4. Add a meaningful string as the request body. For example, `Welcome to Choreo`.
     ![Request Payload](../assets/img/get-started/request-payload.png){.cInlineImage-full}
 
-5. Click **Execute**.
-
-6. Notice the string you entered in step 4 in the response. 
-
-Congratulations! You have created a REST API using a template, deployed, and tested the API. 
+5. Click **Execute**. You'll see the string you entered in step 4 above returned as the server response.
 
-Let's look at how you can add integration logic to the API and further develop this API. Follow the steps in the next section.
+Congratulations! You have now tested the REST API and are ready to implement the business logic.
 
-## Develop your REST API
+## Implement the business logic
 
-Developing the API involves writing the business logic for the API. APIs often need to integrate with third-party systems to create a comprehensive integration solution. With Choreo, you can create your integration solution easily with a low-code, pro-code editor, or both and publish your API with a faster time-to-market.
-
-In this guide, we will add a new resource to the API and simple business logic:
+Developing an API involves writing the business logic for it. APIs often need to integrate with third-party systems to create a comprehensive integration solution. With Choreo, you can create your integration solution easily by implementing the business logic using its low-code editor, pro-code editor, or both.
 
 ### Step 1: Develop in low-code
 
-1. Click **Develop** from the left navigation. 
+Follow this procedure to further develop the API by adding a new resource and simple business logic:
```

Comment:
Shall we avoid saying, further develop? as this is the first time we are developing it?